### PR TITLE
doc updates; link to repo, add some refs, styling

### PR DESCRIPTION
**Pull request by pmuellr**
* added a link to the GitHub repo at the top of the doc; for the PDF version
  I generate, I'd like to provide a pointed to the repo at the very top of the
  doc.  This seemed like a good place.

* split the list of automation tasks for Bash targets into an actual list.

* moved "Enter Bash ..." to a new paragraph for emphasis.

* added a "Want to learn more?" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +59,17 @@ You should be able to run `bash-handbook` at the command line now. This will ope
...
+* taking a screenshot and uploading it to a server
+* processing text that may come in many shapes and forms
+* converting files between different formats
+* parsing a program's output
+* the list goes on ...
 
-**Bash** is a Unix shell written by [Brian Fox](https://en.wikipedia.org/wiki/Brian_Fox_(computer_programmer)) for the GNU Project as a free software replacement for the [Bourne shell](https://en.wikipedia.org/wiki/Bourne_shell). It was released in 1989 and has been distributed as the Linux and OS X default shell for a long time.
+Enter **Bash**, our savior.
+
+Bash is a Unix shell written by [Brian Fox](https://en.wikipedia.org/wiki/Brian_Fox_(computer_programmer) for the GNU Project as a free software replacement for the [Bourne shell](https://en.wikipedia.org/wiki/Bourne_shell). It was released in 1989 and has been distributed as the Linux and OS X default shell for a long time.
```
Comment (pmuellr) [inline README.md]: grumble, I screwed this up.  The double right parens at the end of `(computer))` programmer rendered incorrectly in my viewer, so I deleted the last one, looked fine in my viewer, so committed it.  And it broke on GH.  Let me know if you want me to fix/squash/push

Comment (denysdovhan) [inline README.md]: image

Fix this, please. The most of people read this book on GH, so it must looks perfect here.

**Comment by pmuellr** [comment]
I've updated the PR with a commit resolving the items.  When it's good to go, let me know and I'll squash it and force push to my repo, for a clean commit.

**Comment by denysdovhan** [comment]
Yep, looks good to me. Please, squash and I'll merge.

**Comment by pmuellr** [comment]
squashed and merged

**Comment by denysdovhan** [comment]
Awesome! Merged!
Thank you, @pmuellr!

**Pull request merged**